src/main/java/com/google/api/codegen/discovery/config/java/JavaTypeNameGenerator.java

```diff
@@ -29,6 +30,11 @@
   private static final String NON_REQUEST_SUBPACKAGE = "model";
 
   @Override
+  public String getApiVersion(String apiVersion) {
```

Comment:
Nothing changes anywhere else - this is just a convenience for Go since there is a common regex replacement for all API version names.